package.json

```diff
@@ -67,15 +67,17 @@
   },
   "resolutionsPurpose": "This temporarily chooses the right versions of these libs, which are specified in both deck AND deck-kayenta",
   "resolutions": {
+    "@types/angular": "1.6.26",
     "@uirouter/react-hybrid": "1.0.2",
     "@uirouter/react": "1.0.7",
     "@uirouter/angularjs": "1.0.26",
     "@uirouter/core": "6.0.8",
+    "angular": "1.8.0",
+    "cheerio": "=1.0.0-rc.6",
```

Comment:
```suggestion
    "cheerio": "=1.0.0-rc.12",
```

?

Reply:
Similar to `nth-check` beyond `=1.0.0-rc.6` there is a breaking change that was introduced and I'm not sure if `enzyme` is using what got broken or not :(

Reply:
No worries!